pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -527,8 +528,13 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 		} else {
 			reason = fmt.Sprintf("Unknown reboot reason - power failure or crash - at %s",
```

Comment:
You should presumaby make this string depend on powerCycleFile as well.

Reply:
Done.